CMakeLists.txt

```diff
@@ -233,6 +234,13 @@ foreach(FILE_WITH_EXTRA_TARGETS ${FILES_WITH_EXTRA_TARGETS})
     include(${FILE_WITH_EXTRA_TARGETS})
 endforeach()
 
+if (NOT TBB_VERIFY_DEPENDENCY_SIGNATURE AND WIN32)
+    message(WARNING "Verification of a dependency signature during dynamic "
+        "run-time linking is explicitly disabled. This may lead to security "
+        "vulnerabilities. It is recommended to set "
+        "TBB_VERIFY_DEPENDENCY_SIGNATURE to ON (default value)")
+endif()
```

Comment:
Consider adding `else` branch to show that binary signatures are enabled so that users are aware about potential issues. Something like:
```cmake
message(STATUS "oneTBB is configured to verify signatures of dynamic dependencies loaded during run-time.")
```